common/concurrency/limits/src/main/java/io/helidon/common/concurrency/limits/AimdLimitImpl.java

```diff
@@ -103,13 +107,32 @@ Optional<LimitAlgorithm.Token> tryAcquire(boolean wait) {
         }
         if (wait && queueLength > 0) {
             long startWait = clock.get();
+            Span.Builder<?> spanBuilder = null;
+            if (tracer != null) {
+                spanBuilder = tracer.spanBuilder("limits-wait");
```

Comment:
Maybe use the limit type? i.e. `aimd-limit-wait` and `fixed-limit-wait`

Reply:
done.